format-objc-hook

```diff
@@ -23,7 +23,7 @@ function format_objc() {
 	
     if [ $difference -gt 0 ]; then
     	# This is what the dev can run to fixup an individual file
-    	echo "$DIR/format-objc-file.sh '$file' && git add '$file';"
+    	echo "\"$DIR\"/format-objc-file.sh '$file' && git add '$file';"
```

Comment:
I would expect this one to do the right thing since it was already wrapped in quotes, and is being interpolated here in place, not read later like when we're setting up the repo hook